tests/Unit/Constraint/MethodPair/AccessorPair/data/failure/GetPrivateSet.php

```diff
@@ -7,6 +7,7 @@
 
 /**
  * The getter method is private, so it's not possible to select the accessormethod pair
+ * @SuppressWarnings(PHPMD.UnusedPrivateMethod)
```

Comment:
Misschien deze directory excluden in phpmd? Niet echt nodig om je dataset te inspecten.